cms/templates/manage_users.html

```diff
@@ -53,7 +53,7 @@ <h3 class="title">${_("Add a User to Your Course's Team")}</h3>
                   <li class="field text required create-user-email">
                     <label for="user-email-input">${_("User's Email Address")}</label>
                     <input id="user-email-input" class="user-email-input" name="user-email" type="text" placeholder="${_('example: username@domain.com')}" value="">
-                    <span class="tip tip-stacked">${_("Please provide the email address of the course staff member you'd like to add")}</span>
+                    <span class="tip tip-stacked">${_("Provide an email address to add the user as Staff")}</span>
```

Comment:
I don't think this tip is as clear as the original. Perhaps "Provide the email address of the user who you want to add as Staff."